src/components/Pages/ManageWiki/Features/FederatedProperties.vue

```diff
@@ -1,15 +1,14 @@
 <template>
   <v-form ref="form" v-model="valid">
-    <v-card color="#fad1d0">
+    <v-card disabled tile elevation="2" class="padding" outlined>
```

Comment:
Changing this component to be `disabled` is definitely what we want :)

However I don't think we want to enable tile mode. This causes the corners of the card to become sharp right angles rather than rounded. I realise you were copying exactly what ti says on figma but I figured we should check with the designer because this doesn't match the rest on the components in view.

I think we could also set elevation; however this should probably be set to the same elevation as the other cards on the page (which is 1)

I think we can skip the `class="padding"`

I also think the addition of outlined is great and keeps us in line with the rest of the cards